config/olm/metrics.service.yaml

```diff
@@ -13,11 +12,11 @@ spec:
   type: ClusterIP
   sessionAffinity: None
   ports:
-    - port: 8443
-      name: https
+    - name: https
```

Comment:
Seems to me this is a no change? Do we want to keep the comments and clean this diff?